example/analysis_options.yaml

```diff
@@ -0,0 +1,17 @@
+include: package:lints/recommended.yaml
+
```

Comment:
remove whole file. this way `analysis_options.yaml` from root will be same for whole project.